src/common/mmo.h

```diff
@@ -63,7 +63,7 @@
 // 20120307 - 2012-03-07aRagexeRE+ - 0x970
 
 #ifndef PACKETVER
-	#define PACKETVER 20141022
+	#define PACKETVER 20150513
```

Comment:
Is this intended? Do we fully support that packetver? I think it would be worth doing this in a separate commit in case.